packages/react-devtools-extensions/package.json

```diff
@@ -25,7 +25,6 @@
     "archiver": "^3.0.0",
     "babel-core": "^7.0.0-bridge",
```

Comment:
this is quite possible also redundant now? not sure... but if it was here for jest@23, then it can go 🙂